pontoon/base/urls.py

```diff
@@ -56,7 +56,7 @@
     ),
     # Download translation memory
     url(
-        r"^(?P<locale>[A-Za-z0-9\-\@\.]+)/(?P<slug>[\w-]+)/(?P<filename>.+)\.tmx$",
+        r"^translation-memory/(?P<filename>.+)\.tmx$",
```

Comment:
hey @anujpandey785 
The patch looks good, but I have a small suggestion that potentially can reduce the amount of code.

The regex for this contains a variable that can be removed because it [gets overridden in the view](https://github.com/mozilla/pontoon/pull/1692/files#diff-d964cbd7ee26bf4860da2c98511ccb58R783).
```
r"^translation-memory/(?P<locale>[A-Za-z0-9\-\@\.]+)\.(?P<slug>[\w-]+)\.tmx$",
```

Reply:
Hii @jotes 
Thanks for reviewing the patch .Sorry, I had overlooked the fact that the variable filename is being overridden. Yeah that helps us reducing 5 lines of code I have made the suggested changes , pushing the changes now. Tysm :)

Reply:
@jotes Done 
But as we have changed the url for downloading the translational memory it would have its effect on button for downloading translation memory located in the project dashboard as well , we should change the url there too ?
Also, I am not sure about this but the tests which are failing do we also need to make changes in the test_tmx.py file for both failed cases?

Reply:
I'll take a look at it today :-)

Reply:
Okay, the tests are failing because of the changes in the URLs. I think it's a good idea to use [reverse](https://docs.djangoproject.com/en/3.1/topics/http/urls/#reverse-resolution-of-urls) and replace the hardcoded URLs.

Reply:
> Okay, the tests are failing because of the changes in the URLs. I think it's a good idea to use [reverse](https://docs.djangoproject.com/en/3.1/topics/http/urls/#reverse-resolution-of-urls) and replace the hardcoded URLs.

Thanks for the suggestion. I am working on it.